_episodes/05-conclusions.md

```diff
@@ -69,7 +69,7 @@ to be suitable for use by ESMValTool.
 
 There are lots of resources available for helping you use ESMValTool.
 
-ESMValTool [Discussions page](https://github.com/ESMValGroup/ESMValTool/discussions).
+The ESMValTool [Discussions page](https://github.com/ESMValGroup/ESMValTool/discussions).
```

Comment:
Please remove the dot at the end of this line.

Reply:
done.